<!DOCTYPE html>
<html id=dcfd737dcd4747aaa1551d3cbe4baad5:97449161 data-origid=97449161 class="anndoc" data-anndoc-version="2.0" lang="" xml:lang="" xmlns="http://www.w3.org/1999/xhtml">
	<head>
		<meta charset="UTF-8"/>
		<meta name="generator" content="org.rostlab.relna"/>
		<title>dcfd737dcd4747aaa1551d3cbe4baad5:97449161</title>
	</head>
	<body>
		<article>
			<section data-type="title">
				<h2 id="s1h1">Mutations in the TSC2 gene: analysis of the complete coding sequence using the protein truncation test (PTT).</h2>
			</section>
			<section data-type="abstract">
				<div class="subsections">
					<h3 id="s2h1">Abstract</h3>
					<p id = "s2s1p1">Mutations in the TSC2 gene on chromosome 16p13.3 are responsible for approximately 50 % of familial tuberous sclerosis (TSC). The gene has 41 small exons spanning 45 kb of genomic DNA and encoding a 5.5 kb mRNA. Large germline deletions of TSC2 occur in < 5 % of cases, and a number of small intragenic mutations have been described. We analysed mRNA from 18 unrelated cases of TSC for TSC2 mutations using the protein truncation test (PTT). Three cases were predicted to be TSC2 mutations on the basis of linkage analysis or because a hamartoma from the patient showed loss of heterozygosity for 16p13.3 markers. Three overlapping PCR products, covering the complete coding sequence of mRNA, were generated from lymphoblastoid cell lines, translated into 35S-methionine labelled protein, and analysed by SDS-PAGE. PCR products showing PTT shifts were directly sequenced, and mutations confirmed by restriction enzyme digestion where possible. Six PTT shifts were identified. Five of these were caused by mutations predicted to produce a truncated protein: (i) a sporadic case showed a 32 bp deletion in exon 11, and a mutant mRNA without exon 11 was produced; the normal exon 10 was also spliced out; (ii) a sporadic case had a 1 bp deletion in exon 12 (1634delT); (iii) a TSC2-linked mother and daughter pair had a G -- > T transversion in exon 23 (G2715T) introducing a cryptic splice site causing a 29 bp truncation of mRNA from exon 23; (iv) a sporadic case showed a 2 bp deletion in exon 36; (v) a sporadic case showed a 1 bp insertion disrupting the donor splice site of exon 37 (5007+2insA), resulting in the use of an upstream exonic cryptic splice site to cause a 29 bp truncation of mRNA from exon 37. In one case, the PTT shift was explained by in-frame splicing out of exon 10, in the presence of a normal exon 10 genomic sequence. Alternative splicing of exon 10 of the TSC2 gene may be a normal variant. Three 3rd base substitution polymorphisms were also detected during direct sequencing of PCR products. Confirmed mutations were identified in 28 % of the families studied and on the assumption that half of the sporadic cases should have TSC2 mutations, a crude estimate of the detection rate would be 60 %. This compares favourably with other screening methods used for TSC2, notably SSCP, and since PTT involves much less work it may be the method of choice.</p>
				</div>
			</section>
		</article>
	</body>
</html>