<!DOCTYPE html>
<html id=5bccb887f0b44561a942acec3914d665:99335895 data-origid=99335895 class="anndoc" data-anndoc-version="2.0" lang="" xml:lang="" xmlns="http://www.w3.org/1999/xhtml">
	<head>
		<meta charset="UTF-8"/>
		<meta name="generator" content="org.rostlab.relna"/>
		<title>5bccb887f0b44561a942acec3914d665:99335895</title>
	</head>
	<body>
		<article>
			<section data-type="title">
				<h2 id="s1h1">Clonality analysis of granulocytes and T lymphocytes in healthy females by the PCR-based HUMARA method.</h2>
			</section>
			<section data-type="abstract">
				<div class="subsections">
					<h3 id="s2h1">Abstract</h3>
					<p id = "s2s1p1">Clonality analysis utilizing X-chromosome inactivation has been used in the study of various diseases, including hematological malignancies. The human androgen receptor gene (HUMARA) assay is the newest of such methods, and the majority of the female population can be assessed by this relatively simple procedure. One problem in using these clonality analysis methods, however, is that there may be significant variation in Lyonization in blood cells in normal individuals. To determine the diversity in X-chromosome methylation patterns, which reflect Lyonization, assessed by the HUMARA assay in the supposedly normal population, we analyzed granulocytes and T cells from 97 relatively young (18- to 35-year-old) healthy female volunteers. We found that the methylation patterns in the two HUMARA alleles were distributed even more widely, both in granuloctyes and in T cells, than previously reported with other methods. We also found that the deviation of methylation in granulocytes and T cells was well correlated. Thus, we conclude that appropriate controls from the same individuals, such as T cells in the case of stem cell disorders, should always be employed to conclusively determine whether certain cells of hematopoietic origin are clonal.</p>
				</div>
			</section>
		</article>
	</body>
</html>